src/app/content/highlights/components/Card.tsx

```diff
@@ -127,8 +135,10 @@ const Card = (props: Props) => {
     loginLink={props.loginLink}
     highlight={props.highlight}
     locationFilterId={locationFilterId}
+    hasUnsavedHighlight={hasUnsavedHighlight}
```

Comment:
I think it would be easier to maintain if EditCard would use own selector for obtaining this data

Reply:
I agree. `loginLink` and `user` are also only passed through the `Note` into the `EditNote`, so i assumed that there is a reason why `EditNote` is not being directly connected to the store.

Reply:
I don't think there is any reason for that, maybe @TomWoodward knows?

Reply:
there isn't a particularly good reason, you're free to move selectors down to subcomponents if they're only used there